cli/server/server.go

```diff
@@ -61,6 +61,10 @@ func NewCommands() []cli.Command {
 			Name:  "in, i",
 			Usage: "Input file (stdin if not given)",
 		},
+		cli.StringFlag{
+			Name:  "dump",
+			Usage: "JSON output file with state delta",
```

Comment:
It's not a file, but rather a dir with some special contents, I think we should be more specific describing it here.